src/app/register/components/form-personal/form-personal.component.ts

```diff
@@ -0,0 +1,166 @@
+import { Component, OnInit, forwardRef } from '@angular/core'
+import {
+  FormGroup,
+  ControlValueAccessor,
+  Validator,
+  FormControl,
+  AbstractControl,
+  ValidationErrors,
+  NG_VALUE_ACCESSOR,
+  NG_VALIDATORS,
+  Validators,
+  AsyncValidatorFn,
+  ValidatorFn,
+  FormGroupDirective,
+  NgForm,
+  NG_ASYNC_VALIDATORS,
+} from '@angular/forms'
+import { BaseForm } from '../BaseForm'
+import {
+  TLD_REGEXP,
+  ILLEGAL_NAME_CHARACTERS_REGEXP,
+  URL_REGEXP,
+} from 'src/app/constants'
+import { Observable } from 'rxjs'
+import { RegisterService } from 'src/app/core/register/register.service'
+import { map } from 'rxjs/operators'
+import { OrcidValidators } from 'src/app/validators'
+
+@Component({
+  selector: 'app-form-personal',
+  templateUrl: './form-personal.component.html',
+  styleUrls: ['./form-personal.component.scss'],
+  preserveWhitespaces: true,
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => FormPersonalComponent),
+      multi: true,
+    },
+    {
+      provide: NG_ASYNC_VALIDATORS,
+      useExisting: forwardRef(() => FormPersonalComponent),
+      multi: true,
+    },
+  ],
+})
+export class FormPersonalComponent extends BaseForm implements OnInit {
+  constructor(private _register: RegisterService) {
+    super()
+  }
+
+  emails: FormGroup = new FormGroup({})
+  additionalEmails: FormGroup = new FormGroup({
+    '0': new FormControl('', {
+      validators: [Validators.email, Validators.pattern(TLD_REGEXP)],
+    }),
+  })
+
+  ngOnInit() {
+    this.emails = new FormGroup(
+      {
+        email: new FormControl('', {
+          validators: [
+            Validators.required,
```

Comment:
@leomendoza123 I think it would be a good idea to create a validator for emails which include Validators.required, email and TLD_REGEXP to reuse it whenever is needed

Reply:
That might help sometimes, but also: 

a- keeping the validator on separate instances can help to improve readability on what's been validated for each input 
b- in the future form we would need to use the email validation without it been required 

What do you think?